File: src/main/java/database/DatabaseManager.java
/*
 * Copyright (c) Grigory "vcusnx" Markovich, 2023.
 *
 * The Messenger Project.
 */

package database;

import java.sql.Connection;


public interface DatabaseManager {

    /**
     * @return Connection to the server
     */
    Connection connect();

    /**
     * @return Disconnection from server
     */
    Connection abortConnect();
}

File: app/src/main/kotlin/database/DatabaseManager.kt
/*
 * Copyright (c) Grigory "vcusnx" Markovich, ISLANDS TEAM, 2023.
 *
 *
 */

package database

import java.sql.Connection

interface DatabaseManager {
    /**
     * Connect to database
     */
    fun connect(): Connection

    /**
     * Disconnect from server
     */
    fun disconnect(): Connection
}